Plugins/BenchmarkTool/BenchmarkTool.swift

```diff
@@ -126,10 +132,10 @@ struct BenchmarkTool: AsyncParsableCommand {
     }
 
     func shouldRunBenchmark(_ name: String) throws -> Bool {
```

Comment:
These changes flowed into place when I ran `swiftformat .` using the configuration within the repo. I presumed that was desired when submitting code changes. If not, I can back these out.